idea/testdata/.config/idea/templates/run-configurations/data-node.run.xml.template

```diff
@@ -0,0 +1,33 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="{{ .ConfigName }}" type="Application" factoryName="Application" singleton="true">
+    <envs>
+      <env name="DEVELOPMENT" value="true" />
+{{- if not .UseEnvFile }}
+  {{- range $key, $value := .Env }}
+      <env name="{{ $key }}" value="{{ $value }}" />
+  {{- end }}
+{{- end }}
+    </envs>
+    <option name="WORKING_DIRECTORY" value="$PROJECT_DIR$" />
+    <option name="MAIN_CLASS_NAME" value="org.graylog.datanode.bootstrap.Main" />
+    <module name="data-node" />
+    <option name="PROGRAM_PARAMETERS" value="datanode -f $PROJECT_DIR$/../graylog-project-repos/graylog2-server/misc/datanode.conf" />
```

Comment:
I needed to add the `-np` command flag to prevent pid file conflicts for the data nodes.

```suggestion
    <option name="PROGRAM_PARAMETERS" value="datanode -np -f $PROJECT_DIR$/../graylog-project-repos/graylog2-server/misc/datanode.conf" />
```

Reply:
The `-np` flag will be removed from the Data Node command in https://github.com/Graylog2/graylog2-server/pull/20181. I suggest that we wait until that PR is merged before merging this PR and releasing a new CLI version.